### DISCUSSION TITLE
Apertura Demo Registro

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#207] [Idee e proposte]
Date le richieste di prova per vari istituti che mi scrivono... Non posso rispondere a tutti.... Se per @trinko e per gli altri va bene aprirei una demo del registro su un  hosting...... Creerei una mail dove le persone/scuole mandano le richieste per testare il registro così da creare la demo e mandare le credenziali....

**Comment by trinko** [comment]
Si, certo, sarebbe un aiuto prezioso. Grazie.

**Reply by manuelbarberi** [reply]
https://giuademo.x10.mx/ ho realizzato anche il sito per la demo se non le dispiace

**Reply by manuelbarberi** [reply]
https://giuademo.x10.mx/richiesta-demo/ ecco il link per richiedere le demo...... se lo può inserire da qualche parte.... l'url demo verrà inserito sul sito a breve...

**Reply by trinko** [reply]
Ho inserito il link su GitHun e nel sito della documentazione.

**Reply by manuelbarberi** [reply]
Ok, grazie mille.... qualsiasi cosa mi può contattare

**Reply by scuolagioco** [reply]
@manuelbarberi potrebbe fare anche un servizio di assistenza oppure per qualsiasi cosa posso mandare una mail a richiestedemo@giuademo.x10.mx ?

**Reply by manuelbarberi** [reply]
supporto@giuademo.x10.mx

**Reply by manuelbarberi** [reply]
Ovviamente @scuolagioco se dovessi scrivere qui supporto@giuademo.x10.mx inoltra le credenziali del tuo hosting..... la mia mail è intasata dalle richieste di supporto.... ho creato la mai apposta
